include/open62541/server_pubsub.h

```diff
@@ -179,6 +192,42 @@ struct UA_PubSubConnectionConfig {
 #endif
 };
 
+#ifdef UA_ENABLE_PUBSUB_MONITORING
+
+typedef enum {
+    eMessageReceiveTimeout
```

Comment:
Please rename the enum entry to a unambiguously name and follow the conventions for enums.

Reply:
sorry for the wrong enum convention
called  it 'UA_PUBSUB_MONITORING_MESSAGE_RECEIVE_TIMEOUT'